### PR TITLE
build.go: Switch 32bits compilation to default mode

### PR DESCRIPTION
**Pull request by lelibreauquotidien**
On the compilation in 32 bits system (or with GOARCH=386), build.go returns error "unsupported setting GO386=387. Consider using GO386=softfloat instead."

This PR is tested on grafana compilation.

### CODE REVIEW

**Review by ying-jeanne — APPROVED** [review]
LGTM, since we updated to go 1.16. Pouvez vous signer le "Contributor License Agreement", C'est necessaire pour la contribution maintenant, merci beaucoup.  @lelibreauquotidien

**Review by dsotirakis — APPROVED** [review]

**Review by aknuds1 — COMMENTED** [review]
Is there a good reason to use "softfloat" instead of the default ("sse2")?

File: build.go [hunk 1]
```diff
@@ -343,7 +343,7 @@ func setBuildEnv() {
 		os.Setenv("GOARCH", goarch)
 	}
 	if goarch == "386" {
-		os.Setenv("GO386", "387")
+		os.Setenv("GO386", "softfloat")
```
Comment (aknuds1) [inline build.go:346]: How about dropping this line, so we use the default or someone executing build.go can set `$GO386` themselves to override?

**Review by aknuds1 — CHANGES_REQUESTED** [review]
We've discussed and come to the conclusion we want to go with the $GO386 default instead. This is with performance considerations in mind, if someone wants pre-Pentium4 architecture support they can override by setting $GO386 when building.

File: build.go [hunk 2]
```diff
@@ -343,7 +343,7 @@ func setBuildEnv() {
...
 	}
```
Comment (aknuds1) [inline build.go:347]: Please drop this whole section, so we'll use the default instead ("sse2").

**Review by aknuds1 — CHANGES_REQUESTED** [review]

File: build.go (hunk 2 above)
Comment (lelibreauquotidien) [reply build.go]: OK, this statement is removed in [e091803](https://github.com/lelibreauquotidien/grafana/commit/e09180346ddbae24c17eaafa0220845567709a2b)

**Review by aknuds1 — APPROVED** [review]
LGTM

**Comment by aknuds1** [comment]
Thanks for the fix!

**Pull request merged**